### PR TITLE
Add info on daily reports for server

### PR DESCRIPTION
**Pull request by pblottiere**
Hi,

This PR follows this discussion https://lists.osgeo.org/pipermail/qgis-psc/2018-June/006331.html (with a slight delay).

Actually, the aim is to talk about daily reports generated for the server (OGC tests and performance reports).

This PR is not ready to be merged. In fact, I would like to:
- wait for the dedicated server and deploy WMS CertifSuite and PerfSuite on it
- wait for the WFS CertifSuite to be ready 

Then, I'll update http links accordingly.

Otherwise, I think that 2 subdirectories should be created for OGC tests:
- http://test.qgis.org/ogc_cite/wms_130/
- http://test.qgis.org/ogc_cite/wfs_110/

In the same way as for OGC tests, http://test.qgis.org/perf_test/ directory should probably be splitted:
- http://test.qgis.org/perf_test/ms_perfs/
- http://test.qgis.org/perf_test/graffiti/

I'm not a good writer, so all critics are welcomed :).

### CODE REVIEW

File: source/site/getinvolved/daily_reports/performance.rst [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Performance
+===========
+
+At the moment, two tools are living together to test and generate performance reports.
+
+The first one is `MS-Perfs <https://github.com/camptocamp/ms_perfs>`_ which allows to compare QGIS Server with other map servers (like GeoServer or MapServer). Reports are generated daily and available `here <https://gmf-test.sig.cloud.camptocamp.net/ms_perfs/>`_.
```
Comment (yjacolin) [inline source/site/getinvolved/daily_reports/performance.rst]: * break line to 80 caracters
* `generated daily` -> `generated weekly`

File: source/site/getinvolved/daily_reports/performance.rst [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+The second one is `Graffiti <https://github.com/pblottiere/graffiti>`_, allowing to generate a performance report between several versions of QGIS Server and/or specific project options (parallel rendering effect, trust option, and so on). Reports are also generated daily and available `here <https://gmf-test.sig.cloud.camptocamp.net/ms_perfs/>`_. Note that `QGIS-Server-PerfSuite <http://github.com/Oslandia/QGIS-Server-PerfSuite>`_ project is the deployment infrastructure to run Graffiti in continuous integration.
```
Comment (yjacolin) [inline source/site/getinvolved/daily_reports/performance.rst]: wrong url for the report (link to ms_perfs on Camptocamp server).

File: source/site/getinvolved/daily_reports/performance.rst (hunk 1 above)
Comment (yjacolin) [inline source/site/getinvolved/daily_reports/performance.rst]: the correct url should be the one on test.qgis.org, isn't?

**Comment by pblottiere** [comment]
Thanks for the review @yjacolin!

I updated the documentation accordingly.

**Pull request merged**